File: src/main/java/application/print/SystemClock.java
package application.print;

import java.text.SimpleDateFormat;
import java.util.Calendar;

public class SystemClock implements Clockable {
    private SimpleDateFormat formatter = new SimpleDateFormat("HH:mm");

    @Override
    public String nowHHmm(int minutesAdded) {
        Calendar arrivedTime = Calendar.getInstance();
        arrivedTime.add(Calendar.MINUTE, minutesAdded);
        return formatter.format(arrivedTime.getTime());
    }
}

File: src/main/java/application/print/SystemClock.kt
package application.print

import java.text.SimpleDateFormat
import java.util.*

class SystemClock : Clockable {
    private val formatter = SimpleDateFormat("HH:mm")

    override fun nowInHHmm(addMinutes: Int): String {
        val now = Calendar.getInstance()
        now.add(Calendar.MINUTE, addMinutes)
        return formatter.format(now.time)
    }
}